tst/DataChannelFunctionalityTest.cpp

```diff
@@ -508,6 +508,96 @@ TEST_F(DataChannelFunctionalityTest, createDataChannel_PartialReliabilityOrdered
     freePeerConnection(&answerPc);
 }
 
+TEST_F(DataChannelFunctionalityTest, createDataChannel_DataChannelMetricsTest)
+{
+    RtcConfiguration configuration;
+    PRtcPeerConnection offerPc = NULL, answerPc = NULL;
+    PRtcDataChannel pOfferDataChannel = nullptr, pAnswerDataChannel = nullptr;
+    SIZE_T datachannelLocalOpenCount = 0, msgCount = 0;
+    RemoteOpen remoteOpen{};
```

Comment:
Should this then be initialized to 0 if you are using {} syntax?

Reply:
Not necessary. Default constructor is called anyways. So, it should be fine.